space/internal/components/aclnotifications/notifications.go

```diff
@@ -71,6 +75,8 @@ func (n *aclNotificationSender) Run(ctx context.Context) (err error) {
 }
 
 func (n *aclNotificationSender) Close(ctx context.Context) (err error) {
+	n.mx.Lock()
```

Comment:
I guess we don't need a mutex here

Reply:
I've said that before also

Reply:
We already revised approach a little bit, so some changes will follow